util.go

```diff
@@ -230,8 +230,8 @@ func btsEqualFold(s, p []byte) bool {
 
 	for i := 0; i < n; i++ {
 		x := m + (i << 3)
-		av := *(*uint64)(unsafe.Pointer(&s[x]))
-		bv := *(*uint64)(unsafe.Pointer(&p[x]))
+		av := uint64(*(*byte)(unsafe.Pointer(&s[x])))
```

Comment:
Looks like it is not correct replacement? Its a casting of `byte` value to `uint64`? Or am I misunderstood this?

Reply:
I mean, is it the same as just doing `uint64(p[x])`?

Reply:
Hm, yeah, probably this can be reduced to `uint64(p[x])`. I will double check this a bit later 👌

Reply:
I mean, it can't  – prev implementation was comparing all 8 bytes, this change will compare only one :)
I believe as an extreme thing we may moreover replace this `btsEqualFold` and `strEqualFold` to the stdlib alternatives – there are not so much difference in performance in cases where these functions used (mostly for checking short strings):

```
BenchmarkBtsEqualFold/websocket#0-12                            213459022                5.58 ns/op            0 B/op          0 allocs/op
BenchmarkBtsEqualFold/upgrade#1-12                              134671761                8.86 ns/op            0 B/op          0 allocs/op
BenchmarkBtsEqualFold/rnd_eq_512#2-12                           23062928                52.0 ns/op             0 B/op          0 allocs/op
BenchmarkBtsEqualFold/rnd_ineq_512_256#3-12                     40835487                28.1 ns/op             0 B/op          0 allocs/op

BenchmarkBtsEqualFold/websocket#0_std-12                        43955514                26.4 ns/op             0 B/op          0 allocs/op
BenchmarkBtsEqualFold/upgrade#1_std-12                          56361006                19.6 ns/op             0 B/op          0 allocs/op
BenchmarkBtsEqualFold/rnd_eq_512#2_std-12                         722638              1595 ns/op               0 B/op          0 allocs/op
BenchmarkBtsEqualFold/rnd_ineq_512_256#3_std-12                  1471777               806 ns/op               0 B/op          0 allocs/op
```

That is, we never use them on some long strings.

Reply:
Oh, looks good. So I will update method with funcs mentioned by you.